src/App.js

```diff
@@ -1,22 +1,26 @@
 import React from 'react';
+import { Provider } from 'react-redux';
 import {
   BrowserRouter as Router,
   Routes,
   Route,
 } from 'react-router-dom';
+import store from './redux/configureStore';
 import Books from './components/Books';
 import Categories from './components/Categories';
 import Nav from './components/Nav';
 import './App.css';
 
 const App = () => (
```

Comment:
- [ ] Kindly consider styling the app as the Zeplin design so that it looks clean and professional

|Your app|Zeplin design|
|-|-|
|![Screenshot_20221112_223338](https://user-images.githubusercontent.com/25227022/201495386-6f9e2480-a758-4345-8a5e-6810a80ef66a.png)|![Screenshot_20221110_210233](https://user-images.githubusercontent.com/25227022/201495394-6d995205-768d-4ee2-a7cd-e6fe95e1f2ca.png)|